config/config.yml

```diff
@@ -176,14 +176,23 @@ arisa:
 
         ----
         Restricted by PrivacyModule ??[~arisabot]??
-      allowedEmailRegex:
-        - '^(mailer-daemon|postmaster|nobody|noreply|no-reply|hostmaster|usenet|news|webmaster|www|abuse|noc|security|info|support|uucp|ftp|news|admin|root)@.*'
-        - '.*@(mojang.com|microsoft.com|minecraft.net|zendesk.com)$'
```

Comment:
Have removed the leading `^` and trailing `$` from these patterns because the patterns have to match the entire email address anyway so these meta characters seem redundant.